addons/mail/static/src/models/composer_view.js

```diff
@@ -958,7 +873,7 @@ registerModel({
          * @return {boolean}
          */
         _computeHasSuggestions() {
-            return this.mainSuggestions.length > 0 || this.extraSuggestions.length > 0;
+            return this.mainSuggestables.length > 0 || this.extraSuggestables.length > 0;
```

Comment:
from what the compute does, shouldn't the field be called `hasSuggestables`?

Reply:
I renamed the lists to keep "suggestion" in them.